pkgs/by-name/ob/objfw/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  autoconf,
+  autogen,
+  automake,
+  clang,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "objfw";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    owner = "ObjFW";
+    repo = "ObjFW";
+    rev = "refs/tags/1.1.7-release";
+    hash = "sha256-0ylG/2ZSO3b8zdh6W9QJH5OJW9V344CCik1DduV5mhI=";
+  };
+
+  nativeBuildInputs = [
+    clang
+    automake
+    autogen
+    autoconf
+  ];
+
+  configurePhase = ''
+    runHook preConfigure
+    ./autogen.sh
+    ./configure --prefix=$out --without-tls
```

Comment:
Remove this and add `--without-tls` to `configureFlags`
Prefix among other flags are added by mkderivation so the overriding whole phases should be avoided